data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,24 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+historical_RAPTOR_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") 
```

Comment:
Could you ensure all data frames have names that are all in lowercase? i.e. `_raptor_` and not `RAPTOR`. You'll need to:

* recreate the `data/*.rda` files and delete the previous ones
* Update `NEWS.md` accordingly